client/app/dbaas/logs/welcome/logs-welcome.html

```diff
@@ -38,8 +38,8 @@ <h5 data-translate="logs_welcome_kibana_supported"></h5>
                 data-translate="logs_welcome_try">
             </a>
             <div class="text-center mt-4">
-                <span ng-if="ctrl.TARGET !== 'CA'">
-                    <a class="oui-button_link p-2" ng-href="{{ctrl.urls.productURL}}" target="_blank" data-translate="logs_welcome_product_page"></a> | 
+                <span ng-if="ctrl.region !== 'CA'">
```

Comment:
Not yours but missing a `data-` prefix.

Reply:
Can be fixed in another PR :), not the subject here :)